### PR TITLE
feat: Added lightsail alarms

### LINKED ISSUE #769: AWS::Lightsail::Alarm

### PR DESCRIPTION
**Pull request by amanenk**
#### Summary

closes #769 

---

Use the following steps to ensure your PR is ready to be reviewed

- [x] Read the [contribution guidelines](../blob/main/CONTRIBUTING.md) 🧑‍🎓
- [x] Run `go fmt` to format your code 🖊
- [x] Lint your changes via `golangci-lint run` 🚨 (install golangci-lint [here](https://golangci-lint.run/usage/install/#local-installation))
- [x] Update or add tests. Learn more about testing [here](https://docs.cloudquery.io/docs/developers/sdk/testing) 🧪
- [x] Update the docs by running `go run ./docs/docs.go` and committing the changes 📃
- [x] If adding a new resource, add [relevant Terraform files](../terraform) in a separate PR 📂
- [x] Ensure the status checks below are successful ✅

### CODE REVIEW

File: resources/services/lightsail/gen.hcl [hunk 1]
```diff
@@ -5,23 +5,95 @@ add_generate     = true
 description_modifier "remove_read_only" {
   words = ["  This member is required."]
 }
+#
+#resource "aws" "lightsail" "instances" {
```
Comment (hermanschaaf) [inline resources/services/lightsail/gen.hcl:9]: Why is this being commented out?

**Review by hermanschaaf — APPROVED** [review]

File: resources/services/lightsail/gen.hcl (hunk 1 above)
Comment (hermanschaaf) [reply resources/services/lightsail/gen.hcl]: Ah okay, yeah makes sense. With the new version you should be able to generate without problems :)

**Comment by hermanschaaf** [comment]
Not sure why the integration tests are failing 🤔

**Review by shimonp21 — APPROVED** [review]

File: resources/services/lightsail/alarms.go [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+				Resolver:    schema.PathResolver("MonitoredResourceInfo.Arn"),
+			},
+			{
+				Name:        "monitored_resource_info_name",
+				Description: "The name of the Lightsail resource being monitored",
+				Type:        schema.TypeString,
+				Resolver:    schema.PathResolver("MonitoredResourceInfo.Name"),
+			},
+			{
+				Name:        "monitored_resource_info_resource_type",
```
Comment (bbernays) [inline resources/services/lightsail/alarms.go:97]: `monitored_resource_resource_type`

**Pull request merged**